### PR TITLE
fix: don't try to upload all the files as `/`

### PR DESCRIPTION
**Pull request by ovesh**
fixes VH2-5

### CODE REVIEW

File: tests/test_file_util.py [hunk 1]
```diff
@@ -9,7 +9,7 @@ def test_gcs_cp():
     gcs_client = storage.Client()
     gcs_client.f_orig_bucket = gcs_client.bucket
     dest_bucketname = "bucket1"
-    dest_url = f"gs://{dest_bucketname}/path1/path2/file2.txt"
```
Comment (ovesh) [inline tests/test_file_util.py:12]: This was a bug in the test that allowed the resulting bug to creep through.

**Review by arostamianfar — APPROVED** [review]
Thanks for the fix! BTW, I just realized that our integration tests don't actually use `--output` (they check the output based on the Cromwell's locations). Could you please add an issue for adding it?

**Review by mwkyuen — APPROVED** [review]
LGTM!

**Comment by ovesh** [comment]
Created https://linear.app/dg-eng/issue/VH2-35

**Pull request merged**